src/main/java/org/datadog/jmxfetch/reporter/Reporter.java

```diff
@@ -70,7 +70,7 @@ public void sendMetrics(LinkedList<HashMap<String, Object>> metrics, String inst
             String[] tags = Arrays.asList((String[]) metric.get("tags")).toArray(new String[0]);
 
             // StatsD doesn't support rate metrics so we need to have our own aggregator to compute rates
-            if (!"gauge".equals(metricType)) {
+            if ("counter".equals(metricType)) {
```

Comment:
I am worried that this breaks backward compatibility with configuration that did not have `counter` properly spelt (`!"gauge".equals` versus `"counter".equals`).  What do you think?